Dockerfile

```diff
@@ -5,5 +5,7 @@ RUN ssh-keygen -q -t rsa -N '' -f id_rsa
 RUN mkdir /code
 WORKDIR /code
 ADD requirements.txt /code/
+ADD requirements-dev.txt /code/
 RUN pip install -r requirements.txt
+RUN pip install -r requirements-dev.txt
```

Comment:
Try to minimize number of "RUN" commands. Each such command adds an overlay to the docker FS.